app/controllers/saved_pages_controller.rb

```diff
@@ -1,14 +1,19 @@
 class SavedPagesController < ApplicationController
   include AuthenticatedApiConcern
 
-  before_action :check_saved_page_exists, only: %i[destroy]
+  before_action :check_saved_page_exists, only: %i[destroy show]
 
   # GET /api/saved_pages
   def index
     @saved_pages = SavedPage.where(oidc_user_id: user.id)
     render_api_response(saved_pages: present_pages(@saved_pages))
   end
 
+  # GET /api/saved_pages/:page_path
+  def show
+    render_api_response(saved_page: present_page(saved_page))
```

Comment:
If you implement the `SavedPage#to_hash` method, this can become
```ruby
render_api_response(saved_page: saved_page.to_hash)
```

Reply:
Ok let's roll that conversation into this one: https://github.com/alphagov/account-api/pull/69/files#r633509206